hbase-protocol/src/main/protobuf/Client.proto

```diff
@@ -509,12 +511,15 @@ message RegionActionResult {
 message MultiRequest {
   repeated RegionAction regionAction = 1;
   optional uint64 nonceGroup = 2;
+  // Moved this to RegionAction in HBASE-8458. Keep it for backward compatibility. Need to remove
+  // it in the future.
   optional Condition condition = 3;
 }
 
 message MultiResponse {
   repeated RegionActionResult regionActionResult = 1;
-  // used for mutate to indicate processed only
+  // Moved this to RegionActionResult in HBASE-8458. Keep it for backward compatibility. Need to
+  // remove it in the future.
```

Comment:
Do these fields get updated still or just abandoned?

Reply:
For backward compatibility, we will use these fields.